tests/integration/ceph_smoke_test.go

```diff
@@ -92,6 +92,8 @@ func (s *SmokeSuite) SetupSuite() {
 		Mons:                      3,
 		SkipOSDCreation:           false,
 		EnableAdmissionController: true,
+		ConnectionsEncrypted:      true,
+		ConnectionsCompressed:     false,
```

Comment:
Can we edit the nightly job running against master to validate the compression scenario?